kolibri/plugins/learn/assets/src/views/TopicsPage/index.vue

```diff
@@ -453,16 +459,29 @@
         return this.resourcesDisplayed.length < this.resources.length;
       },
       topics() {
-        return this.contents.filter(content => content.kind === ContentNodeKinds.TOPIC);
+        return this.contents
+          .filter(content => content.kind === ContentNodeKinds.TOPIC)
+          .filter(t => t.children && t.children.results.length)
+          .map(t => {
+            let topicChildren = t.children ? t.children.results : [];
```

Comment:
Looks like the ternary check here will never evaluate to anything except `t.children.results` which should always be something that has a `.length`  because of the filter so maybe could remove the ternary?